### PR TITLE
New component ClosableItem and misc improvements

### PR DESCRIPTION
**Pull request by samuelg0rd0n**

### CODE REVIEW

File: src/components/common/FormControl/FormControlTypes.tsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { ReactElement } from 'react'
```
Comment (FilipPyrek) [inline src/components/common/FormControl/FormControlTypes.tsx:1]: For files where are stores types, we usually use file name `types.ts` or `types.tsx`. The name of the component is mentioned in parent folder name.

Comment (samuelg0rd0n) [reply src/components/common/FormControl/FormControlTypes.tsx]: Renamed.

**Review by FilipPyrek — CHANGES_REQUESTED** [review]
Add `.DS_Store` files to .gitignore please

**Review by FilipPyrek — APPROVED** [review]
💜

**Pull request merged**